### PR TITLE
Update MEMBER_REPRESENTATION.md

### PR DESCRIPTION
**Pull request by jorydotcom**
Enhances document to include audit of W3C account + groups & clarifies this list. partially addresses #173

### CODE REVIEW

File: MEMBER_REPRESENTATION.md [hunk 1]
```diff
@@ -1,27 +1,59 @@
-# Member Representation
+# OpenJS Foundation Representatives
 
-The following document lists OpenJS Foundation members and their representation to any standards organizations:
+The following document lists OpenJS Foundation's memberships and liaison relationships, along with the community members designated to participate on behalf of the Foundation:
+
+* [Ecma International]
+  * [TC39] Delegates
+    * Guy Bedford
+    * Jordan Harband
```
Comment (ljharb) [inline MEMBER_REPRESENTATION.md:8]: "delegates" isn't the right word to use here - the other three are delegates of OpenJS, but i'm an invited expert who also happens to be in the standards group.

Comment (jorydotcom) [reply MEMBER_REPRESENTATION.md:8]: Maybe I'll remove "Delegates" in these cases then.

**Review by ljharb — APPROVED** [review]

File: MEMBER_REPRESENTATION.md [hunk 2]
```diff
@@ -1,27 +1,59 @@
...
 * [Unicode MessageFormat Working Group]
   * Eemeli Aro
 * [W3C]
-  * Brian Kardell
-  * Jory Burson (AC)
-  * Michael Champion
-  * Tobie Langel
+  * [Web-interoperable Runtimes Community Group]
+    * Jory Burson
+    * Robin Ginn
```
Comment (ljharb) [inline MEMBER_REPRESENTATION.md:22]:
```suggestion
    * Robin Ginn
    * Jordan Harband
```
i participate in wintercg whenever i can

**Review by tobie — APPROVED** [review]

File: MEMBER_REPRESENTATION.md [hunk 3]
```diff
@@ -1,27 +1,59 @@
...
+  * [Interledger Payments Community Group]
+    * Kris Borchers
+  * All Affiliated Accounts
+    * Richard Gibson 
+    * Brian Kardell
+    * Jory Burson (AC Rep.)
+    * Michael Champion
+    * Tobie Langel (Alt AC Rep.)
+    * Anne-Gaelle Colom
+    * Richard Gibson
```
Comment (VoltrexKeyva) [inline MEMBER_REPRESENTATION.md:40]: Richard Gibson is added two times here.

Comment (ljharb) [reply MEMBER_REPRESENTATION.md:40]:
```suggestion
    * Richard Gibson 
    * Brian Kardell
    * Jory Burson (AC Rep.)
    * Michael Champion
    * Tobie Langel (Alt AC Rep.)
    * Anne-Gaelle Colom
```

**Review by ljharb — APPROVED** [review]
assuming the two suggestions are accepted

File: MEMBER_REPRESENTATION.md (hunk 2 above)
Comment (Ethan-Arrowood) [reply MEMBER_REPRESENTATION.md:22]: Me too! `* Ethan Arrowood`

File: MEMBER_REPRESENTATION.md [hunk 4]
```diff
@@ -1,27 +1,61 @@
...
 
-The following document lists OpenJS Foundation members and their representation to any standards organizations:
+The following document lists OpenJS Foundation's memberships and liaison relationships, along with [Working Group Members] and community members who participate in these organizations. An asterisk (*) denotes delegates of the OpenJS Foundation. Where applicable, organizational affiliation has been added in parens.
+
+* [Ecma International]
+  * [TC39] 
+    * Guy Bedford* 
+    * Jordan Harband (Invited Expert)
+    * Lea Verou* 
+    * Richard Gibson*
```
Comment (eemeli) [inline MEMBER_REPRESENTATION.md:10]:
```suggestion
    * Richard Gibson* 
    * Eemeli Aro (Mozilla)
```

File: MEMBER_REPRESENTATION.md [hunk 5]
```diff
@@ -1,27 +1,62 @@
...
-The following document lists OpenJS Foundation members and their representation to any standards organizations:
+The following document lists OpenJS Foundation's memberships and liaison relationships, along with [Working Group Members] and community members who participate in these organizations. An asterisk (*) denotes delegates of the OpenJS Foundation. Where applicable, organizational affiliation has been added in parens.
+
+* [Ecma International]
+  * [TC39] 
+    * Guy Bedford* 
+    * Jordan Harband (Invited Expert)
+    * Lea Verou* 
+    * Richard Gibson* 
+    * Eemeli Aro (Mozilla)
```
Comment (ljharb) [inline MEMBER_REPRESENTATION.md:11]:
```suggestion
    * Eemeli Aro (Mozilla)
    * Gus Caplan*
```
cc @devsnek

**Pull request merged**